returnn/forward.py

```diff
@@ -61,6 +62,8 @@ def __init__(
         :param cpu_rqmt: job cpu requirement
         """
         self.returnn_config = returnn_config
+        if model_checkpoint is None:
+            assert not eval_mode, "Eval requires a checkpoint"
```

Comment:
why? does task "eval" not work with random_model_init?

Reply:
eval requires to set an epoch, 0 needs to be set so RETURNN does not want to load a model and then it crashes in `init_network_from_config` line 1114.